mongodbatlas/resource_mongodbatlas_alert_configuration.go

```diff
@@ -670,11 +680,17 @@ func flattenAlertConfigurationThresholdConfig(m *matlas.Threshold) []interface{}
 	return []interface{}{}
 }
 
-func expandAlertConfigurationNotification(d *schema.ResourceData) []matlas.Notification {
+func expandAlertConfigurationNotification(d *schema.ResourceData) ([]matlas.Notification, error) {
 	notifications := make([]matlas.Notification, len(d.Get("notification").([]interface{})))
 
 	for i, value := range d.Get("notification").([]interface{}) {
 		v := value.(map[string]interface{})
+		if v1, ok := v["interval_min"]; ok && v1.(int) > 0 {
+			typeName := v["type_name"].(string)
+			if strings.EqualFold(typeName, "pager_duty") || strings.EqualFold(typeName, "ops_genie") || strings.EqualFold(typeName, "victor_ops") {
```

Comment:
Add `pager_duty`, `ops_genie` and `victor_ops` as constants.

Reply:
Also, I see that `type_name` is not validating the possible values, can we do that instead of doing this during execution?

Reply:
Understood, will add the possibly values and the constants